src/main/java/org/opentripplanner/routing/edgetype/Timetable.java

```diff
@@ -139,6 +139,8 @@ public TripTimes getNextTrip(State s0, ServiceDay serviceDay, int stopIndex, boo
         /* Search at the state's time, but relative to midnight on the given service day. */
         int time = serviceDay.secondsSinceMidnight(s0.getTimeSeconds());
         // NOTE the time is sometimes negative here. That is fine, we search for the first trip of the day.
+        // Adjust for possible boarding time TODO: This should be included in the trip and based on GTFS
+        time += s0.getOptions().getBoardTime(this.pattern.mode);
```

Comment:
Is the adjustment really this simple? Or should it be addition / subtraction and getBoardTime / getAlightTime, depending on whether we're called for depart / arrive and from forward / reverse search?